controllers/book.controller.js

```diff
@@ -37,6 +39,7 @@ module.exports.addBook = (req, res) => {
 module.exports.deleteBook = (req, res) => {
     Book.deleteOne({ _id: req.params.id })
         .then((mres) => {
+            Logger.addLog('books',`${req.userData.firstname} ${req.userData.lastname} deleted ${req.body.title}`);
```

Comment:
Log reads like 'Zachary Baldwin deleted undefined'.

This is because we are not passing anything in the body for this route. So what you may need to do is change this route to first find a book using the `Book.find({ _id: req.params.id })` function, that way we can get access to all the attributes, including the title, before deleting it. You can store the title in a variable and then create the log using that variable.

Reply:
testing

Reply:
<img width="887" alt="Screen Shot 2022-10-07 at 9 10 15 AM" src="https://user-images.githubusercontent.com/52636356/194573592-73176076-95b2-408c-9350-96aeb3632f89.png">

Reply:
resolved